Source/VCGeneration/VCGen.cs

```diff
@@ -17,13 +17,14 @@ namespace VC
 
   public class VCGen : ConditionGeneration
   {
+    public CancellationToken CancellationToken { get; }
```

Comment:
Is this needed?  It would be inherited, no?

Reply:
Correct. I forgot to remove it.